### DISCUSSION TITLE
Simple pagination

### DISCUSSION THREAD

**Discussion by mattusik** [rappasoft/laravel-livewire-tables#989] [Feature Requests]
You are using laravel `paginate` method in `executeQuery`, unfortunately under the hood `select count(*) as aggregate from tableName` is executed all the time. I'm facing huge performance issues because of this in huge tables. Use `simplePaginate` instead of `paginate` would solve my problems. 

I was thinking about something like this:
```
public function configure(): void {
    $this->setPerPageAccepted([5, 10])
         ->setPerPage(5)
      👉 ->setPaginationMode('simple') 👈
         ->setPrimaryKey('user_login_histories.id')
         ->setSearchLazy();
}
```

I know that simplePaginate will support only prev & next links, but that is completely ok.

**Comment by rappasoft** [comment]
Added for next release: https://github.com/rappasoft/laravel-livewire-tables/commit/8eacdfbb6c46c93ebf3d4c3ee352ac582d64cbb1

**Reply by mattusik** [reply]
You are awesome! 
Works perfectly now, even for tables with more than 25 mil records.

Thank you so much 😇
